src/vnsw/agent/vrouter/ksync/windows/vnswif_listener.cc

```diff
@@ -18,37 +18,32 @@
 #include <vrouter/ksync/interface_ksync.h>
 #include "vnswif_listener.h"
 
-VnswInterfaceListenerWindows::VnswInterfaceListenerWindows(Agent *agent) :
-	VnswInterfaceListenerBase(agent) {
+VnswInterfaceListenerWindows::VnswInterfaceListenerWindows(Agent *agent)
+	: VnswInterfaceListenerBase(agent) {
+	/* TODO(sodar): Implement */
```

Comment:
either implement this or change it to `\\ TODO(WINDOWS)` with nice comment

Reply:
This should really have a ticket number